src/qrcore.cpp

```diff
@@ -130,34 +130,32 @@ QRCore::~QRCore() {
 }
 
 QString QRCore::cmd(const QString &str) {
-    std::string tmpstr = str.toStdString();
-    const char* cmd = tmpstr.c_str();
-   // const char *cmd = (const char *)str.toUtf8();
+    QByteArray cmd = str.toUtf8();
     //r_cons_flush();
-    char *res = r_core_cmd_str (this->core, cmd);
-    QString o = (res && *res)? QString::fromUtf8(res): QString();
-    free (res);
+    char *res = r_core_cmd_str (this->core, cmd.constData());
+    QString o = QString(res ? res : "");
+    //r_mem_free was added in https://github.com/radare/radare2/commit/cd28744049492dc8ac25a1f2b3ba0e42f0e9ce93
+    r_mem_free(res);
     return o;
 }
 
-bool QRCore::loadFile(QString path, uint64_t loadaddr=0LL, uint64_t mapaddr=0LL, bool rw=false, bool va=false, int bits = 0, int idx, bool loadbin) {
+bool QRCore::loadFile(QString path, uint64_t loadaddr=0LL, uint64_t mapaddr=0LL, bool rw=false, int va=0, int bits = 0, int idx, bool loadbin) {
     RCoreFile *f;
     if (va==0 || va == 2)
         r_config_set_i (core->config, "io.va", va);
     // NO ONE KNOWS WHY THIS IS FIXING A SEGFAULT. core->file should have already a proper value. Pancake dixit
     //core->file = NULL;
     // mapaddr = 0LL;
-    char *paz = (char*)path.toStdString().c_str();
-    //printf ("FILE OPEN (%s)\n", paz);
-    f = r_core_file_open(core, path.toUtf8(), rw?(R_IO_READ|R_IO_WRITE):R_IO_READ, mapaddr);
+    printf ("FILE OPEN (%s)\n", path.toUtf8().constData());
+    f = r_core_file_open(core, path.toUtf8().constData(), rw?(R_IO_READ|R_IO_WRITE):R_IO_READ, mapaddr);
```

Comment:
Which coding style are we gonna use in iaito? Default from qtcreator? The same in r2? Define an uncrustify profile?